mantidproject/mantid
# Indirect ConvFit - Enable delta function

The Delta Function can be used with the StretchedExpFT mode of ConvFit

**To test:**

Interfaces->Indirect->DataAnalysis->ConvFit
Input: 
[ConvFit.zip](https://github.com/mantidproject/mantid/files/806928/ConvFit.zip)
Sample: `26176_red`
Resolution: `26173_res`
Select Mode `StretchedExpFT`
Ensure if `plot guess` is checked a guess is shown on the miniplot.
Ensure under `Delta Function` the `use` checkbox is enabled
Check `use delta function`
press run


Fixes #16568.

_Release Notes have been updated_

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
